public/locales/en/translation.json

```diff
@@ -140,7 +140,7 @@
     "Disposition": "Disposition",
     "Divide timeline into a number of equal length segments": "Divide timeline into a number of equal length segments",
     "Divide timeline into a number of segments with the specified length": "Divide timeline into a number of segments with the specified length",
-    "Divide timeline into segments with randomized durations and gaps between sergments, in a range specified in seconds with the correct format.": "Divide timeline into segments with randomized durations and gaps between sergments, in a range specified in seconds with the correct format.",
```

Comment:
Im ok with this change. The other changes are too small to be worth it (changing english strings will trigger re-translations for all other languages, so i want to avoid that unless really necessary)

Reply:
Translations will not disappear, just translators will be asked to re-check them ("Needs editing")

Reply:
Yes but it still needs work times languages, so i’d rather avoid for such tiny changes

Reply:
What can I say? Your project, your rules.

Reply:
I removed these (3) changes. You can merge.

Reply:
thanks! I have an app store release coming up soon so i won't merge right now, but after that ill merge it 👍